CONTRIBUTING.md

```diff
@@ -12,6 +12,10 @@ This doc is for contributors to Temporal Server (hopefully that's you!)
 * [Protocol buffers compiler](https://github.com/protocolbuffers/protobuf/) (only if you are going to change `proto` files):
   - Install on macOS with `brew install protobuf`.
   - Install on Ubuntu with `sudo apt install protobuf-compiler`.
+* [Temporal CLI tctl](https://github.com/temporalio/tctl):
+  - Install with `brew install tctl`
+  - Or download it from here https://github.com/temporalio/tctl
+
```

Comment:
You have added make target to install it using go also. I think it worth to mention it here.

Reply:
added